src/contracts.ts

```diff
@@ -1,9 +1,9 @@
 import { FeeAmount } from '@uniswap/v3-sdk'
 import { Contract, providers, Signer } from 'ethers'
-import { ERC20 } from 'vanilla-contracts/typechain/vanilla_v1.1/ERC20'
-import { ERC20__factory } from 'vanilla-contracts/typechain/vanilla_v1.1/factories/ERC20__factory'
-import { VanillaV1Router02__factory } from 'vanilla-contracts/typechain/vanilla_v1.1/factories/VanillaV1Router02__factory'
-import { VanillaV1Router02 } from 'vanilla-contracts/typechain/vanilla_v1.1/VanillaV1Router02'
+import { MockERC20 as ERC20 } from '@vanilladefi/trade-contracts/typechain/vanilla_v1.1/MockERC20'
```

Comment:
Not sure about this, but is `MockERC20` replacement for `ERC20` for now and supposed to be used like this @JaniAnttonen?

Reply:
Nnope, regular ERC20 contract should be found in there still :)

Reply:
Ok reverted, but ERC20 and factory are not present in npm package. Only present in local setup of contracts repo 🤔.

Reply:
Got it, typechains are not updated on releases in contracts repo because `postinstall` script does not run on npm package step!